From b8bbedf7507aba48258685d07d314ee391876cf9 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Thu, 09 Jan 2014 18:26:34 +0000
Subject: Forgotten header update

This was supposed to be in the previous commit.
---
diff --git a/src/disks/gduformatvolumedialog.h b/src/disks/gduformatvolumedialog.h
index 62a43ef..3721fbb 100644
--- a/src/disks/gduformatvolumedialog.h
+++ b/src/disks/gduformatvolumedialog.h
@@ -19,7 +19,9 @@ void     gdu_format_volume_dialog_show         (GduWindow    *window,
                                                 UDisksObject *object);
 void     gdu_format_volume_dialog_show_for_xid (UDisksClient *client,
                                                 gint          xid,
-                                                UDisksObject *object);
+                                                UDisksObject *object,
+                                                GCallback     finished_cb,
+                                                gpointer      cb_data);
 
 G_END_DECLS
 
--
cgit v0.9.2

